swap_meet/item.py

```diff
@@ -1,2 +1,12 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category = "", condition=0):
+        self.category = category 
+        self.condition = condition 
+    
+    def __str__(self):
+        return "Hello World!"
+    
+    def condition_description(self):
+        rating = self.condition 
+        description = ["very gross", "Not that great", "It's okay...", "gently used", "pretty good", "really great!"]
```

Comment:
Good idea using a list to find the appropriate description based on the index position!